gopowerstore_types_test.go

```diff
@@ -47,3 +47,8 @@ func TestAPIError_VolumeAttachedToHost(t *testing.T) {
 	apiError := NewVolumeAttachedToHostError()
 	assert.True(t, apiError.VolumeAttachedToHost())
 }
+
+func TestAPIError_VolumeDetachedFromHost(t *testing.T) {
+	apiError := NewVolumeAttachedToHostError()
+	assert.True(t, apiError.VolumeDetachedFromHost())
```

Comment:
Can a testcase be added for negative scenario as well?

Reply:
Positive and negative test cases are there Niranjan, one is `TestAPIError_VolumeAttachedToHost` and another one `TestAPIError_VolumeDetachedFromHost`, but, both do the same thing for now.